jest.setup.js

```diff
@@ -24,7 +31,13 @@ beforeEach(() => {
   // jest.clearAllMocks(); // Use this function inside your test if you need to reset mocks
   jest.restoreAllMocks(); // Use this function inside your test if you need to reset mocks and restore existing functionality
 
-  config.mocks['$store'] = { getters: { 'i18n/t': jest.fn() } };
+  config.mocks['$store'] = {
+    getters: {
+      'i18n/t':    jest.fn(),
+      'prefs/get': jest.fn()
```

Comment:
Global mocks are just required for the i18n as it's an uncommon pattern.
We should not add anything here, unless is strictly required by every component, which is not the case.

Reply:
I'd argue prefs is common too, its just we don't have enough tests that use it - as we write more tests we're going to end up mocking it in every tests rather that using the same pattern as i18n/t